### PR TITLE
backup and restore jobs

### PR DESCRIPTION
**Pull request by agalitsyn**

### CODE REVIEW

File: resources/app.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ hooks:
             restartPolicy: OnFailure
             containers:
               - name: bootstrap
-                image: stolon-bootstrap:0.0.1
+                image: stolon-bootstrap:0.0.5
```
Comment (sofuture) [inline resources/app.yaml]: should be `:latest`

File: resources/app.yaml [hunk 2]
```diff
@@ -69,4 +69,4 @@ hooks:
             restartPolicy: OnFailure
             containers:
               - name: bootstrap
-                image: stolon-uninstall:0.0.1
+                image: stolon-uninstall:0.0.5
```
Comment (sofuture) [inline resources/app.yaml]: should be `:latest`

File: resources/backup.yaml [hunk 3]
```diff
@@ -10,11 +10,8 @@ spec:
       containers:
       - name: backup
         imagePullPolicy: Always
-        image: apiserver:5000/stolon-backup:0.0.1
-        command: ["/backup.sh"]
-        env:
-          - name: STOLON_BACKUP_DB
-            value: "{{STOLON_BACKUP_DB}}"
+        image: apiserver:5000/stolonctl:0.0.5
```
Comment (sofuture) [inline resources/backup.yaml]: should be `:latest`

**Comment by sofuture** [comment]
I think images need different tags in resources, but otherwise looks good.

**Comment by agalitsyn** [comment]
ref https://github.com/gravitational/stolon/pull/10

**Comment by sofuture** [comment]
:shipit:

**Comment by agalitsyn** [comment]
Can't avoid bash scripts here, because somehow we should map environment variables for `stolonctl` with variables provided by k8s

**Comment by sofuture** [comment]
I think you can make secrets any environment variable name you want:

```
      env:
        - name: NEW_VARIABLE_NAME
          valueFrom:
            secretKeyRef:
              name: mysecret
              key: old_variable_name
```

**Pull request merged**